client/src/containers/Settings/Settings.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { Component, ChangeEvent } from 'react'
+
+import Radio from '../../components/Radio'
+import { Settings as SettingsType } from '../../types/Settings'
+import { Difficulties } from '../../types/Difficulties'
+import './Settings.scss'
+
+const difficulties = [
+  { value: Difficulties.EASY, label: 'Facile' },
+  { value: Difficulties.MEDIUM, label: 'Intermédiaire' },
+  { value: Difficulties.HARD, label: 'Difficile' }
+]
+const roundsToWin = [3, 5, 10]
+const cardSpeeds = [1.5, 2, 2.5]
```

Comment:
For speed values, could we have an input number instead of select ?

Reply:
✅